kcidb_io/schema/v4.py

```diff
@@ -735,7 +735,7 @@ def inherit(data):
             build['comment'] = build.pop('description')
 
     # Inherit tests
-    for test in data.get('tests', []):
```

Comment:
`inherit` function has a couple of `test` variable. Should that be changed to `run`?